source/material_model/steinberger.cc

```diff
@@ -796,6 +692,13 @@ namespace aspect
                                 "prescribed " + Utilities::int_to_string(material_file_names.size()) + " material data files, but there are " +
                                 Utilities::int_to_string(this->n_compositional_fields()) + " compositional fields."));
 
+        if (material_file_names.size() == this->n_compositional_fields() + 1)
+          has_background = true;
+        else
+          has_background = false;
+
+        first_composition_index = 0;
```

Comment:
You added this field but it is hard-coded to 0 for now? Please leave at least a comment.

Reply:
Thanks @tjhei - I've added a comment. This bit of the code is part of a larger restructuring that will hopefully make the lookups easier to use.